web/pdf_viewer.js

```diff
@@ -655,6 +655,9 @@ class PDFViewer {
   async getAllText() {
     const texts = [];
     const buffer = [];
+    const options = {
+      disableNormalization: false,
+    };
```

Comment:
Do we actually need this, since that's the default value and we've unit-tests that'll fail if we ever change the default value in the API?

Reply:
My idea was to add it just for a matter of documentation, but I can add a comment.